content/docs/tips-and-tricks/azure-guide.md

```diff
@@ -0,0 +1,348 @@
+{...}
+```
+
+### Create a Key Vault and permit the AKS cluster to access it
+
+```bash
+# create Azure Key Vault
+az keyvault create --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-kv" --location "EastUS"
+{...}
+
+# get our AKS cluster's Object ID
+az aks show --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster" --query "identityProfile.kubeletidentity.objectId" --output tsv
+<YourAKSClusterObjectID>
+
+# set policy
+az keyvault set-policy --name "bank-vaults-test-kv" --object-id <YourAKSClusterObjectID> --secret-permissions all --key-permissions all --certificate-permissions all
+{...}
+```
+
+### Create Storage account and a Container
+
+```bash
+# create storage account
+az storage account create \
+  --name "bankvaultsteststorage" \
+  --resource-group "bank-vaults-test-rg" \
+  --location "EastUS" \
+  --sku "Standard_RAGRS" \
+  --kind "StorageV2"
+{...}
+
+# get storage account key
+az storage account keys list --account-name "bankvaultsteststorage" --query "[0].value" --output tsv
+<YourStorageAccountKey>
+
+# create container
+az storage container create \
+    --name "bank-vaults-test-container" \
+    --account-name "bankvaultsteststorage"
+{...}
+```
+
+## Step 2: Install Bank-Vaults components
+
+You are will [install](https://bank-vaults.dev/docs/operator/#deploy-operator) the [Vault Operator](https://github.com/bank-vaults/vault-operator), [install](https://bank-vaults.dev/docs/mutating-webhook/deploy/#deploy-the-mutating-webhook) the mutating [Webhook](https://github.com/bank-vaults/vault-secrets-webhook) on the AKS cluster you just created, prepare a `Vault` custom resource, and apply it so it would use Azure resources for authentication, and for storing credentials and data.
+
+### Install Vault Operator
+
+```bash
+# install Vault Operator
+helm upgrade --install --wait vault-operator oci://ghcr.io/bank-vaults/helm-charts/vault-operator
+```
+
+### Install Vault Secrets Webhook
+
+```bash
+# create a new namespace and install the Vault Secrets Webhook in it
+kubectl create namespace vault-infra
+kubectl label namespace vault-infra name=vault-infra
+
+helm upgrade --install --wait vault-secrets-webhook oci://ghcr.io/bank-vaults/helm-charts/vault-secrets-webhook --namespace vault-infra
+```
+
+### Start a pre-configured Vault instance
+
+Create a `cr-azure.yaml` resource definition file and copy-paste this content below into it. Replace `<YourStorageAccountKey>`, `<YourTenantId>`, `<YourAppRegistrationObjectId>`, `<YourAppRegistrationClientSecret>`, `<YourSubscriptionId>` and `<YourAKSClusterObjectID>` with the values acquired above, and also the `spec.unsealConfig.azure.keyVaultName`, `spec.config.storage.azure.accountName`, `spec.config.storage.azure.container` fields if you used other names for these Azure resources than in this guide.
```

Comment:
```suggestion
Create a `cr-azure.yaml` resource definition file as defined below. Replace `<YourStorageAccountKey>`, `<YourTenantId>`, `<YourAppRegistrationObjectId>`, `<YourAppRegistrationClientSecret>`, `<YourSubscriptionId>` and `<YourAKSClusterObjectID>` with the values acquired in the previous steps. 
Make sure to also update `spec.unsealConfig.azure.keyVaultName`, `spec.config.storage.azure.accountName`, `spec.config.storage.azure.container` fields in case other names were used for Azure resources compared to this guide.
```